### PR TITLE
Adding glue, group edits on docker_cmd

### PR DESCRIPTION
**Pull request by smithjd**

### CODE REVIEW

File: r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd [hunk 1]
```diff
@@ -5,6 +5,7 @@ We always load the tidyverse and some other packages, but don't show it unless w
 library(tidyverse)
 library(DBI)
 library(RPostgres)
+library(glue)
```
Comment (jimtyhurst) [inline r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd]: I like that we are using `glue`.

Comment (maryannet) [inline r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd]: please add glue

Comment (deemuni) [inline r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd]: I like glue too

File: r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd [hunk 2]
```diff
@@ -19,20 +20,29 @@ The `docker version` command returns the details about the docker daemon that is
...
 # Naming containers `cattle` for throw-aways and `pet` for ones we treasure and keep around.  :-)
 ```
 
+Remove `cattle` if it exists.
+
 Submit the command constructed above:
 ```{r, eval=TRUE, echo=TRUE}
 system2("docker", docker_cmd, stdout = TRUE, stderr = TRUE)
 ```
+POssible errors here: if docker isn't ready, you'll see errors.
```
Comment (deemuni) [inline r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd]: Correction: "Possible" with lower case o

Comment (jimtyhurst) [inline r-database-docker/04-docker-setup-postgres-connect-with-r.Rmd]: Typo: `POssible` => `Possible`.

**Pull request merged**